bot.py

```diff
@@ -12,6 +14,30 @@
 # target_channel_id = 722609125950750771 # testing
 activity = True
 
+@bot.event
+async def on_message(message):
+    if message.author == client.user:
+        return
+    match = re.search((?:\b|\s+)(?:ope)(?:\s+|\b), message.content.lower)
```

Comment:
I feel your regex can be simplified—we don't need \s because it counts as a non word so \b automatically handles it, I'm not sure we need the non capturing groups either.

Reply:
If you look at this example, https://regexr.com/5791o, it does the same thing I guess.